### PR TITLE
Mingshan/Size op

### PR DESCRIPTION
**Pull request by mingshan-wang**
`Size` op is required for GNMT training.

### CODE REVIEW

**Comment by mingshan-wang** [comment]
TESTNOW

**Comment by mingshan-wang** [comment]
TESTNOW

File: src/ngraph_builder.cc [hunk 1]
```diff
@@ -2309,6 +2309,34 @@ static Status TranslateSigmoidOp(
...
+  // Size has an attribute to specify output, int32 or int64
+  ng::element::Type type;
+  TF_RETURN_IF_ERROR(TFDataTypeToNGraphElementType(dtype, &type));
+
+  auto ng_input_shape = ng_input->get_shape();
+
+  int32 result = 1;
+  for (auto dim : ng_input_shape) {
+    result *= dim;
+  }
```
Comment (sayantan-nervana) [inline src/ngraph_builder.cc]: We could use int64/long here.
tensor sizes might be very big, eg: 128*64*64*512 etc

Comment (mingshan-wang) [reply src/ngraph_builder.cc]: Yes, I will make this change now.

**Pull request merged**